helpers/control.py

```diff
@@ -329,6 +330,9 @@ def stop_tempesta(self):
         tf_cfg.dbg(3, '\tStoping TempestaFW on %s' % self.host)
         cmd = '%s/scripts/tempesta.sh --stop' % self.srcdir
         self.node.run_cmd(cmd, timeout=30, err_msg=(self.err_msg % 'stop'))
+        self.oops.update()
+        if self.oops.warn_count("Oops") > 0:
```

Comment:
False positives happen here. `DmesgFinder.update()` will return full tests log from the begging of the tests. If oops happens once all following tests will be failed. Maybe `update()` should return updates from beginning of the current tests?

If a lot oopses happen, dmesg log may be overflowed thus `DmesgFinder.update()` will get empty string. It's false negative.

Reply:
Update checks only messages after tempesta start